doc/denite.txt

```diff
@@ -96,8 +96,8 @@ EXAMPLES						*denite-examples*
 	\ ['rg', '--files'])
 
 	" Change mappings.
-	call denite#custom#map('_', "\<C-j>", 'move_to_next_line')
-	call denite#custom#map('_', "\<C-k>", 'move_to_prev_line')
+	call denite#custom#map('_', '<C-j>', 'move_to_next_line')
```

Comment:
Is this doc change correct? At least I must add the backslash to make remapping work.